Update/hero

Woozl opened the pull request:
![image](https://user-images.githubusercontent.com/16181779/228897300-d85d4903-7aed-4dff-954d-4bc39d583f1e.png)

Woozl committed:
add title to hero

Woozl committed:
use `box-decoration-break: clone`

suejinkim20 reviewed (approved):
it looks great on the research groups! for the project pages, some of the titles extend past the hero image area. if we're not ready to put these on the project pages, maybe we can have a prop that defines whether the title appears in the hero or in the page (similiar to the way it displays when there is no hero image)

or, adjusting the styles for the hero to accomdate longer titles is still an option! if you go to either the develop or staging site and add "/projects" to the end of the url, you'll see a full list of projects and their titles so you'll have an idea of how long some of the titles are.

I think the longest title is `A Multi-decadal Coastal Water Level Model Reanalysis for Coastal Inundation and Flood Risk Assessment (Reanalysis)`

<img width="1084" alt="Screenshot 2023-04-03 at 10 51 44 AM" src="https://user-images.githubusercontent.com/74567117/229546689-da17ba84-049f-4a52-82b9-73f8764850d5.png">

Woozl committed:
use sx instead of css modules

Woozl committed:
switch to flow layout so heading text can increase size of hero

using the negative margin full bleed trick: https://piccalil.li/tutorial/creating-a-full-bleed-css-utility/

this trick requires the use of 100vw, so overflow-x: hidden was applied to the root so a horizontal scrollbar doesnt appear

Woozl commented:
![image](https://user-images.githubusercontent.com/16181779/229598105-4d446d86-41f9-40a4-8e95-295f6f94fa7d.png)

Switched to flow layout so the title can resize the hero image if necessary. This is the same "full bleed" issue we ran into on helx. I'm using this code:

```css
width: 100vw;
margin-left: calc(50% - 50vw);
```

because of the `100vw`, `overflow-x: hidden` is put on the `html` tag to prevent the horizontal scrollbar. If we ever need horizontal scrolling, we'll have to think of a different solution, but for now this fixes it without changing the html structure significantly.
